checks/2A4BEE.yaml

```diff
@@ -0,0 +1,61 @@
+id: "2A4BEE"
+name: SAPHanaSR global.ini file configuration is properly done
+group: ini_files
+description: |
+  SAPHanaSR global.ini file configuration is properly done
+remediation: |
+  ## Abstract
+  This Trento check verified if SAPHanaSR global.ini file configuration is properly done.
+  ## Remediation
+  Edit the global.ini configuration and make changes as per the best practice guide
+  ## References
+  Azure:
+    - https://learn.microsoft.com/en-us/azure/sap/workloads/sap-hana-high-availability?tabs=lb-portal%2Csaphanasr#tabpanel_3_saphanasr
+    
+  AWS:
+    - https://docs.aws.amazon.com/sap/latest/sap-hana/sap-hana-on-aws-implementing-python-hook-saphanasr-in-sles.html
+  
+  GCP:
+    - https://cloud.google.com/solutions/sap/docs/sap-hana-ha-config-sles#enable-hana-hadr-provider-hook
+    
+  SUSE / KVM: 
+            
+    - https://documentation.suse.com/sbp/sap-15/html/SLES4SAP-hana-sr-guide-PerfOpt-15/index.html#id-1.10.11.4
+                  
+  Nutanix:        
+                  
+    - https://documentation.suse.com/sbp/sap-15/html/SLES4SAP-hana-sr-guide-PerfOpt-15/index.html#id-1.10.11.4
+severity: critical
+metadata:
+  target_type: cluster
+  cluster_type: 
+    - hana_scale_up
+  architecture_type: 
+    - classic
+  hana_scenario:
+    - performance_optimized
+    - cost_optimized 
+
+facts:
+  - name: ini_files
+    gatherer: ini_files@v1
+    argument: global.ini
+
+values:
+  - name: execution_order
+    default: 1
+  - name: path
+    default: "/usr/share/SAPHanaSR/"
```

Comment:
nit:
Do we need this final slash in `/usr/share/SAPHanaSR`? In the docs we don't have it

Reply:
In the hana nodes it is also ending with a trailing / and in the documentation guide aswell there is a trailing /.

https://documentation.suse.com/sbp/sap-15/html/SLES4SAP-hana-sr-guide-PerfOpt-15/index.html#id-1.10.11.4

Reply:
ups, I checked the 1st reference, the one from azure, and that doesn't have it:
https://learn.microsoft.com/en-us/azure/sap/workloads/sap-hana-high-availability?tabs=lb-portal%2Csaphanasr#tabpanel_3_saphanasr
And AWS guide as well doesn't have it:
https://docs.aws.amazon.com/sap/latest/sap-hana/sap-hana-pacemaker-sles-hana-control.html#hook_saphanasr
Anyway, not a big deal